docs/v1/install-asciidoctor-linux.sh

```diff
@@ -0,0 +1,210 @@
+#! /bin/bash
```

Comment:
If this file is not needed locally in the p4lang/p4runtime repo, I'd recommend linking to the copy in the p4-spec repo here: https://github.com/p4lang/p4-spec/blob/main/p4-16/spec/install-asciidoctor-linux.sh

Reply:
By linking, I mean adding that link in instructions in docs/v1/README.md

Note that there are other updates that should be made in docs/v1/README.md, e.g. replacing instructions for Madoko with correponding instructions for AsciiDoc.